tests/spec/fs.watch.spec.js

```diff
@@ -64,4 +64,23 @@ describe('fs.watch', function() {
       });
     });
   });
+
+  it('should get a change event when renaming a file', function(done) {
+    var fs = util.fs();
+
+    fs.writeFile('/myfile', 'data', function(error) {
+      if(error) throw error;
+    })
```

Comment:
nit: Missing semi-colon.

Running `npm test` will catch a lot of these issues for you before you submit your PR.